### DISCUSSION TITLE
Empty state illustration SVG

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#1544] [General] [answered]
Hello there,

Can you please guide us on how can we export SVG for empty state illustration? 

Thank you

**Comment by richbachman** [comment]
Hi @ishah-twilio are you looking for an export of the actual empty state illustration, or a guide on how to export the SVG from Figma?

**Reply by ghostling** [reply]
Hi, @ishah-twilio showed me how to export the SVGs from Figma, but even when Figma allows you to do that, the end result is actually a PNG saved as an SVG. 

For example, this one is not a real SVG ([Figma source](https://www.figma.com/file/BPHOD0b4cbSU89PT4gGH7N/Bulk-action?node-id=8%3A34338)):
<img width="1536" alt="Screen Shot 2021-06-10 at 12 33 08 PM" src="https://user-images.githubusercontent.com/2553665/121586320-617b5380-c9e8-11eb-9693-e9217c1292e9.png">


But this one is ([Figma source](https://www.figma.com/file/BPHOD0b4cbSU89PT4gGH7N/Bulk-action?node-id=166%3A0)):
<img width="1536" alt="Screen Shot 2021-06-10 at 12 34 34 PM" src="https://user-images.githubusercontent.com/2553665/121586365-6fc96f80-c9e8-11eb-86df-11ed28f9b91c.png">


These are the ones I'm looking for:
- https://www.figma.com/file/XeA9WNBcoojrdJbiz5Nj5q/Brand-Illustrations?node-id=1%3A85
- https://www.figma.com/file/XeA9WNBcoojrdJbiz5Nj5q/Brand-Illustrations?node-id=8%3A3
- https://www.figma.com/file/XeA9WNBcoojrdJbiz5Nj5q/Brand-Illustrations?node-id=106%3A2
- https://www.figma.com/file/XeA9WNBcoojrdJbiz5Nj5q/Brand-Illustrations?node-id=1%3A112 (for reference, this one is really an SVG)

**Reply by richbachman** [reply]
Hi @ghostling Thanks for the clarification. We'll reach out to the brand team to see why some of those are just images and not SVG. Stay tuned.

**Reply by richbachman** [reply]
Circling back here. Brand and dsys teams will be meeting to figure out the best path forward here.

**Comment by azhengtwilio** [comment]
I was trying to add the svgs and at the same time following the Empty State guideline in the [Paste doc](https://paste.twilio.design/patterns/empty-states/#dos-and-donts), most of the examples are in png, moving forward seems like svgs are better for scaling? Would the guidelines be updated to svg at some point? 

Would be nice to have one source of truth for the svgs illustrations, when I was implementing them and had corrupted svgs, wasn't sure where to go for them and seemed like I had to make some modifications to them too. Just a bit worried that it's gonna be misaligned across teams that are using the same illustrations for empty state

**Comment by SiTaggart** [comment]
Hey @azhengtwilio 

We hear ya, it's kinda painful right now. After the sync with brand, who own the illustrations, we'll hopefully have a better solution.

> Would the guidelines be updated to svg at some point?

Absolutely they will. We're just in the same position as you in reality. SVG are our preference, but we didn't have them at the time of publishing that page.

> Just a bit worried that it's gonna be misaligned across teams that are using the same illustrations for empty state

Yup, a single, central place where you don't need to do any modifications to the asset would be the ideal state we'd like to get to. 🤞🏼

**Reply by ishah-twilio** [reply]
Thank you @SiTaggart.

We greatly appreciate all that DSys is doing to improve the users/customers experience along.  

We were struggling as a team to understand what can we do to solve our immediate need. We added our comments here for documenting purposes. For now, we are using the SVG received from brand team.

**Reply by azhengtwilio** [reply]
Thank you @SiTaggart.

**Comment by kate-porter** [comment]
Hey @ishah-twilio @ghostling @azhengtwilio - we just had a sync with brand to hammer out this process, and moving forward, the DSYS team is going to help prep SVGs for illustrations to be used in Product to ensure they aren't too large. They will then live in a central "Product" folder within Bynder. We'll also work on getting our docs site updated with these refined guidelines.

What's your timeline for needing these SVGs?

**Reply by azhengtwilio** [reply]
We've already implemented some of the Empty state cards with some svgs we found through asking in the slack channel https://twilio.slack.com/archives/C027WETR3/p1623355276070600 and also some from modifying it ourselves. 

I think we'll just create another ticket to update the svgs when it's ready

**Reply by kate-porter** [reply]
Sounds good! We'll create tickets on our side to get these SVGs finalized and then followup with you when they're ready to go.

**Reply by kate-porter** [reply]
[Here's the ticket](https://issues.corp.twilio.com/browse/DSYS-2866) to get these SVGs created for y'all. We'll followup when they are done!

**Comment by kelseycavitt** [comment, marked as answer]
@ishah-twilio @corinne-nagel @azhengtwilio 
I just joined the DSys team and was able to work on the illustrations you need. The SVGs have been added to a Bynder collection called [Product Optimized Illustrations](https://library.twilio.com/collections/view/5E25D9F0-00FF-443B-9F8F395FDE06AEBC/). They have also been updated in the [Brand Illustrations Figma file](https://www.figma.com/file/XeA9WNBcoojrdJbiz5Nj5q/Brand-Illustrations?node-id=1%3A48). Let us know if you have any further questions! Thanks!

**Reply by azhengtwilio** [reply]
Hi @kelseycavitt, we've updated our code to use these illustrations in the Bynder collection, you can see the results here https://console-storybook.netlify.app/?path=/story/applications-iot-emptystatecard--diy. However, I had to explicitly define the sizes(width, height) of these svgs to make them appear smaller which I'd assume we would want them to have the same sizes for all implementations? If that's the case it would be better to set the height and width of the svgs directly to the standardize size
